### DISCUSSION TITLE
how can I use this in  browser ?

### DISCUSSION THREAD

**Discussion by cuivip** [Mogztter/asciidoctor-web-pdf#538] [Q&A]
how can I use this in  browser ?

**Comment by ggrossetie** [comment]
Hello @cuivip 

You can take a look at: https://github.com/Mogztter/asciidoctor-web-pdf/pull/228
This is a draft but the overall idea remains. You need to produce a printable HTML page from an AsciiDoc document, open the result in a new tab or window (https://developer.mozilla.org/en-US/docs/Web/API/Window/open) and then ask your browser to print this page.

Please note that it will only work in modern browsers such as Firefox and Chromium-based browsers like Chrome, Edge.
